rerun_cpp/src/rerun/recording_stream.cpp

```diff
@@ -319,19 +317,21 @@ namespace rerun {
         return status;
     }
 
-    Error RecordingStream::try_set_properties(archetypes::RecordingProperties& properties) const {
+    Error RecordingStream::try_set_name(std::string_view name) const {
         rr_error status = {};
-        this->log_static(PARTITION_PROPERTIES_ENTITY_PATH, properties);
+        this->log_static(
```

Comment:
nit: `this->` for accessing members is imho not very idiomatic, but what is in C++ these days...